### ISSUE TITLE
More incrementality features?

### ISSUE THREAD

**Issue by sfiruch** [arminbiere/gimsatul#4]
Gimsatul would be even more useful if it supported the IPASIR API (or something equivalent).

**Comment by arminbiere** [comment]
I totally agree, but there are many other more important features that are still lacking.  So I keep this on the 'enhancement' stack.
